update.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+read -p "Start installation? (Y/N)? " -n 1 -r
+echo
+case "$REPLY" in 
+  y|Y )
+    tput setaf 4 ; tput bold ;echo "Starting..."; tput sgr0;
+    tput setaf 4 ; tput bold ;echo "Deleting Previous Files"; tput sgr0;
+    rm -r /home/xtreamcodes/iptv_xtream_codes/admin/* 2> /dev/null
+    tput setaf 4 ; tput bold ;echo "Downloading Update"; tput sgr0;
+    apt-get install unzip -y
+    wget https://github.com/Nilbertocs/XtreamUI/archive/master.zip -O /tmp/update.zip -o /dev/null
```

Comment:
This points to the wrong git repo
```suggestion
    wget https://github.com/xtreamui/XtreamUI/archive/master.zip -O /tmp/update.zip -o /dev/null
```